themes/10up-theme/tailwind.config.js

```diff
@@ -44,9 +96,10 @@ module.exports = {
 		},
 	},
 	plugins: [
-		/* Extend with 3rd party tailwind plugins or locally import your own */
-
-		/* Add Tailwind Prose https://tailwindcss.com/docs/typography-plugin */
-		tailwindProse(),
+		/*
+			Extend with 3rd party tailwind plugins or locally import your own
+			All plugins are included by default since they are not output if unused.
+		*/
+		typographyPlugin,
 	],
```

Comment:
@iansvo I see that you have added the other plugins to the package.json. Let's add them here too and just leave commented out

Reply:
I actually had these in from my previous changes, but I removed them in the latest push. 

I think if we're going to include them in package.json it makes sense to just have them loading all the time (except I guess forms). That said, I can't imagine a scenario in which we'd want to inherit form styles over the theme's own styles, but that might be fine for certain types of projects/situations. 

But things like line-clamp and aspect-ratio are so benign it feels like it might be confusing to _not_ include them, since so many examples on the web (Hyper, TailwindUI, and custom stuff) all make use of these. It seems like it would make the set up a bit more beginner friendly, as newbies might not realize these things are normally opt in. I of course don't want to include any unnecessary overhead on any project/tooling, but I think these are harmless and the benefits of having them built in seem worth it from what I can see. 

Does anyone have an actual reason why including these would be a negative, especially if we have documentation that adequately explains them and how they can be removed if you actually want to?

Reply:
I don't think there are any "negative" reasons not to include them. That being said, when it comes to extending core functionality - especially within the context of the `wp-scaffold` the approach has always been "opt-in" which to be honest, is kinda the opposite of what WordPress stipulates as a feature mantra but that's a totally different argument. 

One thing we need to keep in mind is the cognitive load of a feature like this, it takes a different approach in general to building sites, so our first priority should be to provide the option in a way that leaves the decision up to the discipline lead whether or not to use any functionality that is not considered critical. This may not apply to a simple line-clamp but in the past on projects, I've seen FEE, especially less senior engineers be thrown off and confused by far less complicated approaches than the above. 

We also, want to keep surprising the team right? So in my opinion, keep the plugins installed, comment them out like Nikki mentions and we can poll the FEE team to see if they would indeed use / like a feature like this. I have no doubt skilled FEE would consider it a no brainer (hence our debate) but we should always make sure to accommodate those who arent as technically astute as ourselves. 

</Rant>

@iansvo it may be an idea to poll the team now, out of the context of Tailwind to see who is even using `clamp` on projects - that way we can see if 70% of the team use it already, its a no brainer to include it. If only 10% of the team use it, well that means we need to educate, document and leave it up to them to decide if they want to implement in their projects :)